### DISCUSSION TITLE
Create and setting up on-chain DCDAO on Tally

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#127] [Ideas]
Test Demo for playing with:

https://www.tally.xyz/gov/hello-tally/proposal/108334889340808743410529938205527127726421400335266633109162487602226211109557

There will be many use cases for DCDAO, including managing DCDAO Treasury assets and do on-chain airdrops for memberships etc, it can be a supplement or even a replacement for [snapshot](https://snapshot.org/#/darwiniacommunitydao.eth), eliminating some of the execution steps.

Request technical support from @hujw77 @nblogist 

Refer: https://github.com/orgs/dcdao/discussions/87

**Comment by nblogist** [comment]
@hujw77 Aki told me youa re taking care of this pls ping me if you need any help.

**Reply by hujw77** [reply]
You could see todos on https://github.com/dcdao/contracts/issues/4
